neurodamus/connection_manager.py

```diff
@@ -322,7 +322,7 @@ def open_synapse_file(self, synapse_file, edge_population, n_files=1, load_offse
 
     # - override if needed
     def _open_synapse_file(self, synapse_file, pop_name, n_nrn_files=None):
-        logging.info("Opening Synapse file %s, population: %s", synapse_file, pop_name)
+        logging.debug("Opening Synapse file %s, population: %s", synapse_file, pop_name)
```

Comment:
Why `debug`? I've found this info msg very helpful when I want to know the edges files that neurodamus loads without checking `circuit_config.json`.

Reply:
I believe very similar information is printed anyway from another side, hence I decreased this specific line to debug. We can still still see it when debugging with `--debug`

Reply:
Who would print that? I agree with @WeinaJi that this is important information that should be in the logs most of the time, and not buried in debug output. There are other things that are less important in the default logs, IIRC.

Reply:
With new patch:
```
[STEP] Circuit NodeB
[INFO] Processing edge file /home/runner/work/neurodamus/neurodamus/tests/simulations/usecase3/local_edges_B.h5, pop: NodeB__NodeB__chemical
```